v3/testdata/index_scan

```diff
@@ -52,5 +62,16 @@ SELECT * FROM a WHERE y > 1
 4: [gt [2 3]]
 3: [const 1]
 2: [variable a.y]
-1: [scan a] [index-scan a@primary <+0>] [index-join <+1> [6]]
-6: [index-scan a@y_idx <+1>]
+1: [scan a] [index-scan a@primary <+0> [-]] [index-scan a@y_idx <+1> [-]]
+
+prep,memo,search
+SELECT * FROM c WHERE y > 1
+----
+5: [select [1 4]]
+4: [gt [2 3]]
+3: [const 1]
+2: [variable c.y]
+1: [scan c] [index-scan c@primary <+0> [-]] [inner-join [6 8 -]] [inner-join [8 6 -]]
```

Comment:
Notice that select-to-scan creates `[inner-join [6 8 -]]` and join commutativity creates `[inner-join [8 6 -]]`.